staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -213,7 +215,7 @@ type JobSpec struct {
 	Parallelism *int32 `json:"parallelism,omitempty" protobuf:"varint,1,opt,name=parallelism"`
 
 	// Specifies the desired number of successfully finished pods the
-	// job should be run with.  Setting to nil means that the success of any
+	// job should be run with.  Setting to `nil` means that the success of any
```

Comment:
```suggestion
	// job should be run with.  Setting to `null` means that the success of any
```

null is the value in JSON/YAML. Although we are not very consistent putting it within ticks.